tests/foreman/ui/test_host.py

```diff
@@ -2466,3 +2466,134 @@ def test_positive_set_multi_line_and_with_spaces_parameter_value(
         assert host_parameters
         assert param_name in host_parameters
         assert host_parameters[param_name] == param_value
+
+
+class TestHostAnsible:
+    """Tests for Ansible portion of Hosts"""
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier2
+    def test_positive_host_role_information(self):
+        """Assign Ansible Role to a Host and an attached Host group and verify that the information
+        in the new UI is displayed correctly
+
+        :id: 7da913ef-3b43-4bfa-9a45-d895431c8b56
+
+        :assignee: sbible
+
+        :Steps:
+            1. Register a RHEL host to Satellite.
+            2. Import all roles available by default.
+            3. Create a host group and assign one of the Ansible roles to the host group.
+            4. Assign the host to the host group
```

Comment:
Use consistent punctuation at the end of list items. I don't think we have a convention governing this across Robottelo, but within a test, use periods at the end of either all of the steps or none of the steps.